.gitignore

```diff
@@ -120,3 +120,6 @@ venv.bak/
 .mypy_cache/
 *~
 *#
+
+# Anthony's
+test.py
```

Comment:
Just been using this for testing network stuff. Hopefully this is no big deal to add.